crates/nu-parser/src/parser.rs

```diff
@@ -2606,6 +2606,11 @@ pub fn parse_duration_bytes(num_with_unit_bytes: &[u8], span: Span) -> Option<Ex
     let unit_groups = [
         (Unit::Nanosecond, "NS", None),
         (Unit::Microsecond, "US", Some((Unit::Nanosecond, 1000))),
+        (
+            Unit::Microsecond,
+            "\u{039C}S",
+            Some((Unit::Nanosecond, 1000)),
+        ),
         (Unit::Millisecond, "MS", Some((Unit::Microsecond, 1000))),
```

Comment:
Maybe I am pedantic to think that the SI prefixes should only parse if they are in their correct case (which would eliminate potential confusion `m`/`M` etc.)

Reply:
Are you thinking we only accept lower cases and if the input is not, it raises an error, or instead of converting up [uppercase](https://github.com/nushell/nushell/pull/8647/files/3b14756be2e1b5abf986b062013052b35372341c#diff-8c238cdde8119bd1053a4f211e752738c53d8bb10ccd9ab0eef6921be7f5ee50R2605) we convert to lower case?

Reply:
The `into duration`'s `--convert` option only allows for lowercase, so it would make sense to me to unify them so they both only allow lowercase input.

Reply:
While it could be a breaking change for some I think at least for the fractional seconds following SI notation I would embrace the change. 
People might argue about the intervals `>= sec` but I think that should only be driven if a particularly common tool or fle format would prefer uppercase. (bar that I would argue for all duration units lowercase)

Reply:
having them all lowercase wouldn't hurt my feelings.

Reply:
PR updated to force lowercase